.github/workflows/badge.yaml

```diff
@@ -1,6 +1,7 @@
 name: Update Coverage Badge
 
 on:
+  pull_request:
```

Comment:
is there a specific reason we want this to run on pull request? assuming someone makes a bad PR that gets squashed and not merged, the codecov badge will still be updated. Unless codecov ONLY is reading from our main branch. Do you know?

Reply:
It's on pull request because it's best practice to create a new branch when making a change, then creating a pull request on that branch. This way, main is always in a working state.

Once the branch is merged, you can check if it's updating main's coverage.

https://app.codecov.io/gh/liatrio/endybot

Reply:
Sorry it took so long to respond.